code/modules/clothing/suits/armor.dm

```diff
@@ -67,13 +67,22 @@
 /obj/item/clothing/suit/armor/hos/alt
 	name = "armored trenchoat"
 	desc = "A trenchcoat enchanced with a special lightweight kevlar. The epitome of tactical plainclothes."
-	icon_state = "hostrench"
-	item_state = "hostrench"
+	icon_state = "hostrench_open"
+	item_state = "hostrench_open"
 	flags_inv = 0
 	ignore_suitadjust = 0
+	suit_adjusted = 1
 	action_button_name = "Open/Close Trenchcoat"
 	adjust_flavour = "unbutton"
 
+/obj/item/clothing/suit/armor/hos/alt/verb/openjacket(var/mob/user)

```

Comment:
I left this one in because it's the only suit getting adjusted that isn't a child of /suit/storage
I figured it's acceptable because no other armour that I know of gets adjusted